docs/skills/ci.md

```
@@ -2002,3 +2002,37 @@ gh pr view <n> --repo projectbluefin/dakota --json autoMergeRequest \
+    branches: [main, testing, next]
+  merge_group:
+```
+
+(Dakota PR 997, 2026-06-21)
+
+### Junction bumps always cause a cold rebuild of OCI elements (expected)
+
+When `gnome-build-meta.bst` or `freedesktop-sdk.bst` is bumped to a new commit, BST recomputes cache keys for every element downstream. This invalidates all cached Dakota-specific artifacts (`oci/bluefin.bst`, `bluefin/` elements, etc.).
+
+**What "cold rebuild" looks like in logs:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)